README.md

```diff
@@ -1,21 +1,25 @@
-# AmpliPi Home Audio System
+# AmpliPro Home Audio System
 [![Build Status][workflow-badge]][workflow-link] [![Coverage][coverage-badge]][coverage-link] [![GPL License][license-badge]](COPYING)
```

Comment:
Somewhere in this document, we need important links, perhaps right at the top. Links we need are as follows:
1. Online Community (discourse)
2. Latest news and guides (blog)
3. How-to videos (youtube channel)
4. AmpliPro products (www.amplipro.com)

did I miss any other important links?